server/rest/src/main/java/org/hawkular/apm/server/rest/AnalyticsHandler.java

```diff
@@ -524,6 +560,42 @@ public void getTraceCompletionFaultCount(
 
     }
 
+    @POST
+    @Path("trace/completion/faultcount")
+    @Produces(APPLICATION_JSON)
+    @ApiOperation(
+            value = "Get the number of trace instances that returned a fault",
+            response = Long.class)
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Success"),
+            @ApiResponse(code = 500, message = "Internal server error") })
+    public void getTraceCompletionFaultCount(
```

Comment:
Yes, this is a more general change that needs to be made to all GET/POST alternatives. So if ok will raise a separate jira to cover all.